src/app/static-pages/landing-page/sections/news-section/news-section-item/news-section-item.component.html

```diff
@@ -6,8 +6,11 @@
     <i class="fas fa-quote-right quote-end"></i>
   </div>
   <p class="source">{{ item.source }}</p>
-  <a [href]="item.link" target="_blank" class="button button--mysterious-blue">
+  <a [href]="item.link" target="_blank" class="button button--mysterious-blue" *ngIf="item.type!=='audio'">
     <i class="fal fa-external-link-square"></i>
     Beitrag öffnen
   </a>
+  <audio controls *ngIf="item.type==='audio'">
+    <source [src]="item.link" type="audio/mpeg">
+  </audio>
```

Comment:
Könnten wir zukünftig evtl. in ein `ngSwitch` umwandeln - sofern mehrere Typen dazu kommen.
Für den Moment passts aber.